### PR TITLE
Adds new asset metadata example page

### PR DESCRIPTION
**Pull request by iamplugged**
Specs: https://www.figma.com/file/U0awkNQKLDKmOP7k9xiZUQ/WA-Review?node-id=3211%3A18701

To Do:
- [x] Update pin image once available
- [x] Add Google maps API key

Implementation:
image

### CODE REVIEW

**Comment by vpetersson** [comment]
Great stuff, @iamplugged!

File: asset-metadata/src/templates/metadata_headers.html [hunk 1]
```diff
@@ -1,14 +1,77 @@
...
+    <div class="content">
+      <div id="map"></div>
+      <div class="metadata">
+        <div class="header">
+          <img src="{{ url_for('static', filename='images/logo.svg') }}" alt="screenly logo" />
+          <span>Screen Metadata</span>
+        </div>
+        <div class="data">
+          <div class="metadata-item">
+            <label>Host name:</label>
```
Comment (vpetersson) [inline asset-metadata/src/templates/metadata_headers.html:20]: Should be "Hostname"

**Review by vpetersson — CHANGES_REQUESTED** [review]
Minor change, but we also need to add fallbacks if the headers are unavailable.

File: asset-metadata/src/templates/metadata_headers.html (hunk 1 above)
Comment (iamplugged) [reply asset-metadata/src/templates/metadata_headers.html]: Changes done

**Review by vpetersson — APPROVED** [review]

**Pull request merged**